lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -14,7 +14,13 @@
   position: relative;
   width: nx-dropdown-variables.$dropdown-width;
 
-  .nx-icon:not(.nx-icon--colorful, .nx-dropdown-link .nx-icon, .nx-filter-dropdown__reset .nx-icon, .nx-dropdown :is(.disabled, [disabled]) .nx-icon) {
+  .nx-icon:not(
+    .nx-icon--colorful,
+    .nx-dropdown-link .nx-icon,
+    .nx-filter-dropdown__reset .nx-icon,
+    .nx-dropdown :is(.disabled, [disabled]) .nx-icon,
+    .nx-navigation-dropdown .nx-icon
```

Comment:
It is increasingly unfortunate that this bit of code has to be aware of these various special cases, but that's the pattern for the time being.